0001-kaa-protocol/README.md

```diff
@@ -93,40 +106,88 @@ Examples of extension specific paths are:
 /json/status
 ```
 
-### Endpoint-specific and nonspecific extensions
+### Payload format specifier
+Extensions may support multiple payload formats.
+In that case, it is wise to add payload format specifier.
 
-There are two large classes of extensions: those that are endpoint-specific and require an endpoint to operate on, and those that do not. While most extensions are endpoint-specific (data collection, profiling, configuration, etc.), there are extensions that are not (endpoint registration).
+For example, use `/json` for JSON-formatted payload, and `/protobuf/<scheme_id>` for protocol buffers.
```

Comment:
@Slash32 I think we can only define that per extension.

Still, there is little sense in doing so, as they only define JSON now. Once extensions enter the stable state, we won't be able to define new formats without rolling out a new version of protocol extension.